### PR TITLE
peptide-shaker add version 1.15.0

### PR DESCRIPTION
**Pull request by jj-umn**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by druvus** [comment]
It seems that you have the 1.15.0 version both in the sub and main folder. The idea is to create a subfolder for the old recipe, ie 1.14.6 in this case.

**Comment by jj-umn** [comment]
@druvus Within galaxy, I need to reference  particular versions of searchgui and peptide-shaker.  I'd have to find out how galaxy builds an env without a version.  @bgruening Can you enlighten me?

**Comment by bgruening** [comment]
@jj-umn what @druvus means it just the ordering of the recipes I think. The directory structure should be
```
1.14.6
1.x.y
meta.yml (latest)
build.sh (latest)
```

File: recipes/peptide-shaker/1.15.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "PeptideShaker" %}
+{% set version = "1.15.0" %}
```
Comment (bgruening) [inline recipes/peptide-shaker/1.15.0/meta.yaml]: You probably wanted to move `1.14.6` into the `1.14.6` directory, isn't it?

**Pull request merged**